src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  const hasWife = sex === 'm' && 'wife';
+  const hasHusband = sex === 'f' && 'husband';
```

Comment:
You can use only one variable here:
```suggestion
  const partner = sex === 'm'
    ? 'wife'
    : 'husband'
```